app/assets/js/mautic-form-src.js

```diff
@@ -225,6 +225,13 @@ var t,e;t=this,e=function(){"use strict";function t(t,e){var n=Object.keys(t);if
             }
         };
 
+        Form.syncSliderOutputs = function(formId) {
+            var theForm = document.getElementById('mauticform_' + formId);
+            if (!theForm) return;
+            var outputs = theForm.querySelectorAll('output.mauticform-slider-value');
```

Comment:
```suggestion
            const theForm = document.getElementById('mauticform_' + formId);
            if (!theForm) return;
            const outputs = theForm.querySelectorAll('output.mauticform-slider-value');
```
Let's not use `var` anymore.